rootfs/apply.go

```diff
@@ -108,7 +108,14 @@ func ApplyLayerWithOpts(ctx context.Context, layer Layer, chain []digest.Digest,
 		}
 	}
 	return applied, nil
+}
 
+// ApplyLayerToMountWithOpts applies a single layer on top of the given mounts.
+func ApplyLayerToMountWithOpts(ctx context.Context, layer Layer, mounts []mount.Mount, a diff.Applier, applyOpts []diff.ApplyOpt) (bool, error) {
```

Comment:
Since `applyLayerToMount` is not exported should we just export it and change the calling convention to return the `bool` as well rather than adding a wrapper here?